Update Imperial Shipyard

Sabre142 opened the pull request:
New Rumare Village conflict:

![The Elder Scrolls IV Oblivion_2022 02 16-19 55](https://user-images.githubusercontent.com/26516348/154229873-e43e1dd8-e0cc-4d2d-878d-b3aac7940695.png)

Sabre142 committed:
Update Imperial Shipyard

* Remove incompatibility
 - With Let the People Drink. Version 0.4 is now compatible.

* Update cleaning data
 - Version 0.4.

sibir-ine commented on masterlist.yaml:
```diff
@@ -12858,13 +12858,12 @@ plugins:
     dirty:
       - <<: *quickClean
-        crc: 0x33EE0585
+        crc: 0x35FD4202
         util: '[TES4Edit v4.0.4b](https://www.nexusmods.com/oblivion/mods/11536)'
-        itm: 5245
-        udr: 13
+        itm: 5222
+        udr: 52
```
V0.5 was released yesterday & appears to have been run through QAC. However, there appears to be at least one wild edit (NPC `0018B118`), so a message could be added to replace the old cleaning data (sorry, the line deletions are preventing a suggestion), or the old cleaning data can just be removed. The new checksum is 0x09D58350:

```yaml
    msg:
      - <<: *wildEdits
        subs: [ 'Remove NPC **0018B118** from **Imperial Shipyard.esp**.' ]
        condition: 'checksum("Imperial Shipyard.esp", 09D58350)'
```

Sabre142 committed:
Update Imperial Shipyard

* Add incompatibility
 - With New Rumare Village:
   https://www.nexusmods.com/oblivion/mods/51189/

* Add message
 - Wild edit in 0018B118.

* Update cleaning data
 - Version 0.5.
